app/component/itinerary/IndoorRouteInfo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import { FormattedMessage } from 'react-intl';
+import { configShape } from '../../util/shapes';
+import { isKeyboardSelectionEvent } from '../../util/browser';
+import Icon from '../Icon';
+
+export default function IndoorRouteInfo(
```

Comment:
In UI context, Route refers to transit lines such as tram 7 or bus 500.  Users search itineraries, not routes. In finnish we have to share the word Reitti for many purposes but in code we can avoid that.

I have a feeling that dropping Route from the names would  be good:
- IndoorInfo
- IndoorStep
- IndoorSteps or IndoorLeg
- navigator/IndoorButton

etc.